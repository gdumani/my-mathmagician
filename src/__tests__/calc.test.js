import React from 'react';
import renderer from 'react-test-renderer';
import Calc from '../pages/calc';

test('match calc snapshot', () => {
  const tree = renderer.create(<Calc />).toJSON();
  expect(tree).toMatchSnapshot();
});
