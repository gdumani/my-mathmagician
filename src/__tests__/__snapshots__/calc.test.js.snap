exports[`test match calc snapshot 1`] = `
<div
  className="cal-page">
  <h3>
    Lets do some Math!
  </h3>
  <div
    className="calc">
    <span
      data-testid="display">
      
    </span>
    <button
      onClick={[Function]}
      type="button">
      AC
    </button>
    <button
      onClick={[Function]}
      type="button">
      +/-
    </button>
    <button
      onClick={[Function]}
      type="button">
      %
    </button>
    <button
      className="colored"
      onClick={[Function]}
      type="button">
      รท
    </button>
    <button
      onClick={[Function]}
      type="button">
      7
    </button>
    <button
      onClick={[Function]}
      type="button">
      8
    </button>
    <button
      onClick={[Function]}
      type="button">
      9
    </button>
    <button
      className="colored"
      onClick={[Function]}
      type="button">
      x
    </button>
    <button
      onClick={[Function]}
      type="button">
      4
    </button>
    <button
      onClick={[Function]}
      type="button">
      5
    </button>
    <button
      onClick={[Function]}
      type="button">
      6
    </button>
    <button
      className="colored"
      onClick={[Function]}
      type="button">
      -
    </button>
    <button
      onClick={[Function]}
      type="button">
      1
    </button>
    <button
      onClick={[Function]}
      type="button">
      2
    </button>
    <button
      onClick={[Function]}
      type="button">
      3
    </button>
    <button
      className="colored"
      onClick={[Function]}
      type="button">
      +
    </button>
    <button
      className="double"
      onClick={[Function]}
      type="button">
      0
    </button>
    <button
      onClick={[Function]}
      type="button">
      .
    </button>
    <button
      className="colored"
      onClick={[Function]}
      type="button">
      =
    </button>
  </div>
</div>
`;
